scripts/partition_manager/partition_manager.rst

```diff
@@ -15,15 +15,18 @@ The Partition Manager is activated for all multi-image builds, no matter what bu
 Overview
 ********
 
-The Partition Manager script reads a set of Partition Manager configurations.
-Using this information, it deduces the start address and size of each partition.
+The Partition Manager script reads configuration files, each of which is named :file:`pm.yml`.
+Each configuration file names partitions and constrains their placement on flash.
+Partition manager allocates a start address and size to each partition in a way that satisfies these constraints.
```

Comment:
Will fix the capitalization.

Partition Manager is referred to without a definite article throughout this document, though, which makes sense, since it's a proper noun and thus doesn't need it. Examples:

- As output, Partition Manager creates a HEX file [...]
- Partition Manager may implicitly or explicitly assign a HEX file [...]
- Parition Manager stores all variables as target properties [...]
- For this purpose, Partition Manager provides static configuration [...]

Reply:
There are at least as many occurrences with "the", and this is how we usually do it for names like "Manager", "Client" or the like.

Reply:
Then I think there should be a separate commit which makes the style consistent, no?

Reply:
Sure, makes sense to add a commit to fix it everywhere - but we shouldn't introduce wrong usage in new text.